etc/cvmfs/domain.d/osgstorage.org.conf

```diff
@@ -14,28 +14,41 @@ else
 fi
 CVMFS_KEYS_DIR=$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/osgstorage.org
 
-
-CVMFS_EXTERNAL_URL="http://unl-cache.nationalresearchplatform.org:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://stashcache.t2.ucsd.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://its-condor-xrootd1.syr.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-chicago-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-new-york-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://stash-cache.osg.chtc.io:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-gftp2.pace.gatech.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://daejeon-kreonet-net.nationalresearchplatform.org:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-houston-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-sunnyvale-stashcache.t2.ucsd.edu:8000/"
+# OSDF caches used by OSG
```

Comment:
We don't need this comment line in the osg branch, because by definition everything here is used by OSG.

Reply:
done